src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.CoreRT.cs

```diff
@@ -574,31 +568,31 @@ private static unsafe int ComputeThisPtrForDispatchSection(int dispatchCount)
         }
 
         [UnmanagedCallersOnly]
-        internal static unsafe int ABI_QueryInterface(IntPtr ppObject, Guid* guid, IntPtr* returnValue)
+        internal static unsafe int IUnknown_QueryInterface(IntPtr ppObject, Guid* guid, IntPtr* returnValue)
```

Comment:
Nit:
- The first argument name should `pThis`, `this` or `This`. It is the name typically used for this argument in COM. Also, `pp` would suggest that it is a pointer to a pointer which it is not.
- The last two argument names should be `riid` and `ppvObject` to match the names in the C/C++ definition of the method.